truss/getGlobalConfigS3.go

```diff
@@ -0,0 +1,58 @@
+package truss
+
+import (
+	"os"
+	"path"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/mitchellh/go-homedir"
+)
+
+// GetGlobalConfigS3Input input for GetGlobalConfigS3
+type GetGlobalConfigS3Input struct {
+	Bucket string
+	Region string
+	Key    string
+	Role   string
+	Dir    string
+}
+
+// GetGlobalConfigS3 fetch global config from S3 and put it in home dir
+func GetGlobalConfigS3(input *GetGlobalConfigS3Input) error {
```

Comment:
Not quite sure how to test this without mocking the AWS stuff out so much that it's not even really testing anything.

I suppose I could do something like this https://github.com/instructure-bridge/truss-cli/blob/master/truss/getKubeconfigS3_test.go#L12. @jgramoll how are you handling those `TEST_*` credentials?

Reply:
just using a .envrc with
```export TEST_S3_BUCKET=truss-kubeconfig-us-east-2
export TEST_AWS_ROLE=arn:aws:iam::127178877223:role/xacct/ops-admin
export TEST_VAULT_ROLE=admin
```